buttons/buttons.css

```diff
@@ -20,3 +25,40 @@
     padding: 0.5em 1em;
     font-size: 20px;
 }
+
+.burger-btn {
```

Comment:
oh this burger would make me smile all the time now ))))))

Reply:
Actually I saw 2 more same type of burger buttons (1 was even made with a separate jpg of burger).